### PR TITLE
Update CRM grid sample to use IgxGridToolbar

### PR DESCRIPTION
**Pull request by rmkrmk**
This PR removes the toolbar created in the sample and uses the toolbar provided in the IgxGrid.

### CODE REVIEW

**Comment by desig9stein** [comment]
@gedinakova this is the correct contrast color based on the W3C Accessibility standards. you can check it [here](https://webaim.org/resources/contrastchecker/), instead of changing the text I should probably change the logo color.

**Comment by gedinakova** [comment]
@desig9stein It would look different compared to how the Propeller grid used to look. Not sure what should be done here.

File: src/app/grid/grid-crm/grid-crm.component.html [hunk 1]
```diff
@@ -1,206 +1,166 @@
...
-                                    <li class="dd-list__item" *ngFor="let col of cols">
-                                        <igx-checkbox (change)="toggleVisibility(col)" [checked]="col.hidden">{{col.header}}</igx-checkbox>
-                                    </li>
-                                </ul>
+    <igx-grid #grid1 id="grid1" [data]="localData" [height]="'430px'" [width]="'1180px'" [rowSelectable]="true"
+        displayDensity="cosy" rowHeight="50" [allowFiltering]="true"
+        [showToolbar]="true" [columnHiding]="true" columnHidingTitle="Column Hiding"
+        [columnPinning]="true" [exportExcel]="true" [exportCsv]="true" exportExcelText="Export to Excel" exportCsvText="Export to CSV">
+
+        <ng-template igxToolbarCustomContent let-igxGrid="grid">
```
Comment (gedinakova) [inline src/app/grid/grid-crm/grid-crm.component.html]: Remove "let-igxGrid" from the template - it's not necessary.

Comment (rmkrmk) [reply src/app/grid/grid-crm/grid-crm.component.html]: Done.

**Pull request merged**